lib/elixir_internal_certificate_web/views/api/v1/user_registration_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserRegistrationView do
+  use JSONAPI.View, type: "user_auth"
+
+  def fields do
+    [
+      :email,
+      :password
```

Comment:
Are we return the password to the API response, might I know the reason for that @nvminhtue 🤔